### PR TITLE
corrected spellings #35 #34 #32

### PR DESCRIPTION
**Pull request by creatorpiyush**

### CODE REVIEW

**Comment by creatorpiyush** [comment]
i did #35 #34 #32 #37

**Comment by Chaitanyassr** [comment]
@creatorpiyush  Issue number should be in the PR description and not the PR title.

File: src/documents/activities/CauseAndEffect.html.md.handlebars [hunk 1]
```diff
@@ -4,35 +4,39 @@ layout: content
 category: Activities
 ---
 
-This activity demonstrates the potential to have a broader beneficial impact with our everyday actions. The challenge is to define a possible chain of events that could take you from a specific cause to an effect.
+This activity demonstrates the potential to have a broader beneficial impact with our everyday actions. The challenge is
```
Comment (Chaitanyassr) [inline src/documents/activities/CauseAndEffect.html.md.handlebars:7]: Also additional change.

File: src/documents/activities/CauseAndEffect.html.md.handlebars [hunk 2]
```diff
@@ -4,35 +4,39 @@ layout: content
...
+Causes defined in this exercise are everyday tasks such as:
 * share meeting minutes in a form that your colleague using a screen reader can access, or
 * coach an employee in managing a pension.
 
 While the effects might include:
 * grandmother in Ecuador connects with children in LA, or
 * polar bear is saved from ice flow in Alaska.
 
-The effects are as far away from the causes as possible. 
+The effects are as far away from the causes as possible.
```
Comment (Chaitanyassr) [inline src/documents/activities/CauseAndEffect.html.md.handlebars:18]: additional change.

File: src/documents/activities/CauseAndEffect.html.md.handlebars [hunk 3]
```diff
@@ -4,35 +4,39 @@ layout: content
...
 
-Use the accompanying [Cause and Effect card deck](/images/CauseAndEffectCards.pdf) (downloadable pdf), or think up your own examples. These should be as specific and diverse as possible, and the relationship between cause and effect should not immediately be clear.
+Use the accompanying [Cause and Effect card deck](/images/CauseAndEffectCards.pdf) (downloadable pdf), or think up your
```
Comment (Chaitanyassr) [inline src/documents/activities/CauseAndEffect.html.md.handlebars:20]: additional change. A PR should have minimum change for fixing an issue.

File: src/documents/activities/CauseAndEffect.html.md.handlebars [hunk 4]
```diff
@@ -4,35 +4,39 @@ layout: content
...
+own examples. These should be as specific and diverse as possible, and the relationship between cause and effect should
+not immediately be clear.
 
 ## Step 1
 Choose a random card from the Cause deck, then do the same from the Effect deck. Or describe your own cause and effect.
 
 ## Step 2
-Define a chain of effects that takes you all the way from the initial cause to the final effect. Record each effect and map out the chain of effects - allowing branching or multiple paths where they emerge.
```
Comment (Chaitanyassr) [inline src/documents/activities/CauseAndEffect.html.md.handlebars:25]: what does this change is for?

File: src/documents/activities/CauseAndEffect.html.md.handlebars [hunk 5]
```diff
@@ -4,35 +4,39 @@ layout: content
...
+Define a chain of effects that takes you all the way from the initial cause to the final effect. Record each effect and
+map out the chain of effects - allowing branching or multiple paths where they emerge.
 
 ## Why
 
 <ul class="idg-articleContentUseWhyHow">
-<li>
```
Comment (Chaitanyassr) [inline src/documents/activities/CauseAndEffect.html.md.handlebars:30]: additional change.

File: src/documents/activities/CauseAndEffect.html.md.handlebars (hunk 1 above)
Comment (creatorpiyush) [reply src/documents/activities/CauseAndEffect.html.md.handlebars:7]: these are the spelling changes

File: src/documents/activities/CauseAndEffect.html.md.handlebars (hunk 2 above)
Comment (creatorpiyush) [reply src/documents/activities/CauseAndEffect.html.md.handlebars:18]: these are the spelling changes

File: src/documents/activities/CauseAndEffect.html.md.handlebars (hunk 3 above)
Comment (creatorpiyush) [reply src/documents/activities/CauseAndEffect.html.md.handlebars:20]: these are the spelling changes

File: src/documents/activities/CauseAndEffect.html.md.handlebars (hunk 4 above)
Comment (creatorpiyush) [reply src/documents/activities/CauseAndEffect.html.md.handlebars:25]: these are the spelling changes

File: src/documents/activities/CauseAndEffect.html.md.handlebars (hunk 1 above)
Comment (Chaitanyassr) [reply src/documents/activities/CauseAndEffect.html.md.handlebars:7]: Why deleting the 7th line and adding it? I dont think there is any kind of spelling mistake in that.

**Comment by jhung** [comment]
Fixed in PR #40 .

**Pull request closed without merging**